src/webapp/components/date-picker/DatePicker.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import styled from "styled-components";
+import { InputLabel } from "@material-ui/core";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { DatePicker as DatePickerMUI } from "@mui/x-date-pickers/DatePicker";
+import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+import { IconCalendar24 } from "@dhis2/ui";
+
+type DatePickerProps = {
+    id: string;
+    label?: string;
+    value: Date | null;
+    onChange: (value: Date | null) => void;
+    helperText?: string;
+    errorText?: string;
+    disabled?: boolean;
+    error?: boolean;
+    required?: boolean;
+};
+
+export const DatePicker: React.FC<DatePickerProps> = React.memo(
```

Comment:
Do you use React extension? Are components defined as `export const Name = React.memo(...)` visible as `Name` or `Anonymous`?

Reply:
Wow, first time I see this. They are shown as _c, but it's ok if I change the component like this (or if they are not memo):

```typescript
export const Section: React.FC<SectionProps> = React.memo(function Section({
    title = "",
    lastUpdated = "",
    headerButtom,
    hasSeparator = false,
    children,
    titleVariant = "primary",
}) {
    return (
        <SectionContainer $hasSeparator={hasSeparator}>
            {hasSeparator && <Separator />}
            <Header>
                <TitleContainer>
                    {title ? <Title $titleVariant={titleVariant}>{title}</Title> : null}
                    {lastUpdated ? (
                        <LastUpdatedContainer>
                            <LastUpdatedTitle>
                                {i18n.t("Last updated: ", { nsSeparator: false })}
                            </LastUpdatedTitle>
                            {lastUpdated}
                        </LastUpdatedContainer>
                    ) : null}
                </TitleContainer>
                {headerButtom ? <div>{headerButtom}</div> : null}
            </Header>
            <Content>{children}</Content>
        </SectionContainer>
    );
});
``` 

![Screenshot from 2024-07-04 14-48-58](https://github.com/EyeSeeTea/zebra-dev/assets/49402898/1447e96e-d0af-4f5e-811a-4672204bd2a8)
![Screenshot from 2024-07-04 14-49-31](https://github.com/EyeSeeTea/zebra-dev/assets/49402898/07100c82-dd44-4ba5-9c30-6439e2a2101b)